doc/nrf/app_dev/device_guides/nrf54l/kmu_basics.rst

```diff
@@ -35,12 +35,22 @@ Locked keys
 Once provisioned, locked keys are permanently available for use and cannot be deleted without erasing the device.
 For these keys, the revocation policy (RPOLICY) must be marked as ``locked``.
 
+.. _ug_nrf54l_developing_basics_kmu_provisioning_keys:
+
 Provisioning keys for the bootloader
 ************************************
 
 The bootloader can use multiple key generations for image verification (up to three for nRF54L SoCs).
 To safeguard against unauthorized provisioning by attackers, you must :ref:`provision all key generations onto the device<ug_nrf54l_developing_provision_kmu>`.
 
+Make sure to provision the setup’s relevant key sets before any run with bootloaders, including the first boot.
```

Comment:
can reference to that message be added here https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/app_dev/device_guides/nrf54l/fota_update.rst?plain=1#L196 in order to emphasize importance of KMU provisioning? I'm expecting text like:
`otherwise bootloader will not boot FW setup and might take unwanted actions see: the reference. `